### PR TITLE
Ruby OOP Concept practical submission commit-1

### PR DESCRIPTION
**Pull request by adityakansara12**

### CODE REVIEW

File: abstraction.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                                             => These methods' implementation is hidden from the car(object) user.
+=end
+
+class Car
+  def initialize color, type, owner_name1, company
+		@color = color
+		@type = type
+		@owner_name= owner_name1
+		puts "A #{@company} of color: #{@color} and type: #{@type} is purchased by #{@owner_name}"
+	end
```
Comment (dhruvika29kevat) [inline abstraction.rb]: Give proper indentation

File: inheritance.rb [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	end
+
+  attr_accessor :owner_name
+end
+
+class Sedan < Car
+
+  def initialize color, owner_name, company
+		super(color, "SEDAN", owner_name, company)
+	end
```
Comment (dhruvika29kevat) [inline inheritance.rb]: Give proper indentation in all places

**Review by dhruvika29kevat — COMMENTED** [review]
Give proper indentation in all the files and Resolve conflict

**Comment by dhruvika29kevat** [comment]
Give proper indentation in all the files and Resolve conflict

File: inheritance.rb (hunk 2 above)
Comment (adityakansara12) [reply inheritance.rb]: Problem Resolved

File: abstraction.rb (hunk 1 above)
Comment (adityakansara12) [reply abstraction.rb]: Problem Resolved

**Pull request merged**